tools/clang/lib/SPIRV/TypeTranslator.cpp

```diff
@@ -278,6 +278,13 @@ bool TypeTranslator::isSpirvAcceptableMatrixType(QualType type) {
   return isMxNMatrix(type, &elemType) && elemType->isFloatingType();
 }
 
+QualType TypeTranslator::getElementType(QualType type) {
+  QualType elemType = {};
+  (void)(isScalarType(type, &elemType) || isVectorType(type, &elemType) ||
```

Comment:
We are not using the result of this expression. Compilers may generate warnings with some switch turned on (-W unused variable or something). This is to suppress it and also make it clear we are not interested in the result.